### PR TITLE
Annature (Independent Publisher)

### PR DESCRIPTION
**Pull request by iddycol**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

Test_operations

Note:
Delete Endpoint is a Trigger managed internally by the Webhook. But has been tested this webhook via Flow. Rest of the Actions are Tested from the Test Console.

Flow_Operations

Webhook_operations

### CODE REVIEW

File: independent-publisher-connectors/Annature/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      "api_client_id": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true",
+            "tabIndex": 2
+          },
+          "description": "Annature API Client Id",
+          "displayName": "Annature Api Id",
+          "tooltip": "Specifiy Annature Api Client Id"
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Annature/apiProperties.json:13]: @iddycol all files should be free of grammatical errors and typos. in the tooltip you have "Specifiy...", can you please change to "Specify..."

File: independent-publisher-connectors/Annature/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        }
+      },
+      "api_key": {
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Annature/apiProperties.json:16]: @iddycol can you please explain how this connector is using client id and api key?

File: independent-publisher-connectors/Annature/README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# Annature
+
+Annature is an Australia's leading digital signature provider, purpose built for all industries. It's an ISO 27001 certified technology company that builds digital signature, identity verification, and payment collection software.
+
+Consumers from small businesses to enterprises, use our software to manage their digital transactions online. This connectors supports only core operations like Getting Documents, Creating Envelope, Sending to recipients.
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Annature/README.md:5]: @iddycol all files should be free of grammatical errors and typos. in the description you have "...This connectors...", did you meant to say "...This connector..."

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/Annature/apiProperties.json (hunk 2 above)
Comment (nk-gears) [reply independent-publisher-connectors/Annature/apiProperties.json:16]: Annature doesn;t uses oAuth and it uses a ID and Key combination for validation. To make this precise, we have update the file with simple word "Annature Id". Please review now  and let us know if you have any queries.

File: independent-publisher-connectors/Annature/apiProperties.json (hunk 1 above)
Comment (nk-gears) [reply independent-publisher-connectors/Annature/apiProperties.json]: Issue Fixed now.

File: independent-publisher-connectors/Annature/README.md (hunk 3 above)
Comment (nk-gears) [reply independent-publisher-connectors/Annature/README.md]: We have removed the long sentences and updated with simple description. Please review now.

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hello @iddycol,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by Amjed-Ayoub** [comment]
Hello @iddycol,
I hope you are doing well.
Congratulations, the deployment of your connector and documentation page was already completed, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program.
Thank you very much for working with us.

**Review by Amjed-Ayoub — APPROVED** [review]

**Pull request merged**